mmv1/templates/terraform/resource.html.markdown.erb

```diff
@@ -193,6 +193,27 @@ This resource does not support import.
 
 <%= object.name -%> can be imported using any of these accepted formats:
 
+```
+<% import_id_formats_from_resource(object).map{ |id| id.gsub('%', '') }.each do |id_format| -%>
+<%= id_format %>
```

Comment:
Nit: I would consider making this a list (if you haven't already tried). Something like
```
* `<%= id_format %>`
```

As it is now with the single code block, each format doesn't look particularly distinct.

Reply:
That's a good idea, and I agree that it's not distinct from other blocks as it currently is.